pkg/azuredisk/controllerserver.go

```diff
@@ -198,11 +206,11 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	diskName = azureutils.CreateValidDiskName(diskName)
 
 	if resourceGroup == "" {
-		resourceGroup = d.cloud.ResourceGroup
```

Comment:
since only `userAgent` is changed, I think it's ok to use `d.cloud.ResourceGroup`